include/exadg/postprocessor/error_calculation.h

```diff
@@ -52,24 +49,19 @@ struct ErrorCalculationData
   void
   print(dealii::ConditionalOStream & pcout, bool unsteady)
   {
-    print_parameter(pcout, "Calculate error", analytical_solution_available);
-    if(analytical_solution_available == true && unsteady == true)
+    print_parameter(pcout, "Error calculation", unsteady == true && analytical_solution);
+    if(unsteady == true && time_control_data.is_active)
```

Comment:
I am not sure why this is done only for unsteady problems. Another reminder for the future ...

Reply:
This was also not clear for me.